### PR TITLE
Introduce overview doc for all coordinator roles

### PR DESCRIPTION
**Pull request by mxie**
We have a lot of information in many different places. This new doc will
serve as an overview of each role's responsibilities and will eventually
replace `task-descriptions.md`. Details on how to do specific things as
a particular role will be linked to instead of described inline.

### CODE REVIEW

File: overview-of-roles.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+first 1-2 hours each day of the workshop.
+
+**Responsibilities:**
+
+* Set up registration on ticketing platform.
+* Manage waitlist (if configured).
+* Generate reports and guest lists as needed for venue, catering, child care, etc.
+* Coordinate check-in at workshop.
+
+See [How To: Registration](./How To: Registration.md).
```
Comment (edwardloveall) [inline overview-of-roles.md]: I think you meant a different file 🙂

File: overview-of-roles.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+**Time commitment:** 1-2 hours per week (starting 2 weeks before workshop) +
+first and last hours of workshops to meet with parents and sitter.
+
+**Responsibilities:**
+
+* Work with Registration Coordinator to check for childcare requests.
+* Confirm childcare needs with the parents who requested them.
+* Book a sitter (or more than one).
+* Introduce parents and sitter(s) to each other.
+* Follow up with both at the conclusion of workshop.
```
Comment (edwardloveall) [inline overview-of-roles.md]: I think there's a file we can link to for this role that I wrote up?

File: overview-of-roles.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+**Time commitment:** 1-2 hours per week (starting 2-3 weeks before workshop) +
+Friday night of workshop.
+
+**Responsibilities:**
+
+* Keep all flash drives up-to-date with required software (including virtual machine image).
+* Review installation instructions.
+* Serve as main point of contact for questions about software installation.
+
+## Teaching Staff Coordinator
```
[the remark this answers is not in the document]
Comment (mxie) [reply overview-of-roles.md]: I really like Volunteer Coordinator! I think it describes the actual role better.

File: overview-of-roles.md (hunk 2 above)
Comment (mxie) [reply overview-of-roles.md]: I think this stuff is in the task-descriptions.md file, which I plan on removing. Let me do some separate cleanup for childcare-related stuff first and then I'll add the link here.

**Pull request merged**